src/ng-xform/datepicker-field/datepicker-field.component.html

```diff
@@ -0,0 +1,15 @@
+<div *ngIf="!editing" id="{{field.key}}-label">
+  <strong>{{ field.label }}</strong>
+  <p class="text-muted">{{ form?.controls[field.key]?.value | date:'mediumDate':field.locale }}</p>
+</div>
+
+<div *ngIf="editing" id="{{elementId}}-div" class="form-group" [formGroup]="form" [ngClass]="displayFieldCss()">
+  <label for="{{elementId}}">{{field.label}}</label>
+  <input type="text" class="form-control" [formControl]="componentControl" id="{{elementId}}" bsDatepicker
```

Comment:
`bsDatepicker` directive should be right after the `<input`, sou we could easily identify that is a Datepicker and not a text input

Reply:
OK